lib/Model/IMAPMessage.php

```diff
@@ -761,7 +765,7 @@ private function generateTagEntites(array $tags, string $userId): array {
 		$t = [];
 		foreach ($tags as $keyword) {
 			// Map the old $important to $label1 until we have caught them all
-			if ($keyword === '$important') {
+			if ($keyword === '$important' || $keyword === 'important') {
```

Comment:
didn't we want to leave that one out for now until we know *important* is just a remnant from an earlier test?

Reply:
I understood it as the other way around!